### ISSUE TITLE
Error on M1 in memory.py

### ISSUE THREAD

**Issue by ofirgalcon** [tuxudo/memory#1]
I am seeing this error on M1 MBA, not sure if it's important as the module seems to work OK.

``` memory.py Error: Traceback (most recent call last):
  File "/usr/local/munkireport/scripts/memory.py", line 215, in <module>
    main()
  File "/usr/local/munkireport/scripts/memory.py", line 203, in main
    global_ecc_state = ecc_state(info)
  File "/usr/local/munkireport/scripts/memory.py", line 134, in ecc_state
    if array[0]['global_ecc_state'] == 'ecc_enabled' :
KeyError: 'global_ecc_state'
Jan 19 2021 19:12:26 +0000 WARNING: memory.py return code: 1```

**Comment by tuxudo** [comment]
Resolved with PR #2

**Comment by ofirgalcon** [comment]
It doesn't display the error anymore, but now it displays a blank page for both Memory and RAM on M1, works fine on Intel 11.2.3

I think the Memory page issue is a result of update to 5.6.5
